lib/waves.rb

```diff
@@ -0,0 +1,74 @@
+class Waves
+  def initialize
+    @wave1_image = Gosu::Image.new('media/wave1.png')
+    @wave2_image = Gosu::Image.new('media/wave2.png')
+    @wave3_image = Gosu::Image.new('media/wave3.png')
+
+  end
+
+  # apply velocity to the position
+  def update
+    current_time = Gosu::milliseconds()
```

Comment:
we might want to consider providing both `t` and `dt` from `Window` so that we only need to call `Gosu::milliseconds` once.. But for now this is okay